### ISSUE TITLE
Format of the request duration TimeSpan is incorrect

### ISSUE THREAD

**Issue by juliushardt** [Microsoft/ApplicationInsights-Home#115]
The [description of the RequestData duration string](https://github.com/Microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/RequestData.bond#L17) does not match the format which the Application Insights API expects. Right now it describes the format as `d:hh:mm:ss.fffffff`, whereas it should be `d.h:mm:ss.fffffff` (the colon needs to be replaced with a dot), as discussed in [this thread on the MSDN forums](https://social.msdn.microsoft.com/Forums/vstudio/en-US/1c626c8e-0f6a-49a3-9b3c-7fd77fe8d108/field-duration-on-type-requestdata-is-not-a-valid-duration-string-even-though-it-is-valid?forum=ApplicationInsights).

Unfortunately, the API itself returns the same incorrect format in its error messages, which can be confusing:
```
{
  "itemsReceived": 1,
  "itemsAccepted": 0,
  "errors": [
    {
      "index": 0,
      "statusCode": 400,
      "message": "108: Field 'duration' on type 'RequestData' is not a valid duration string.
Expected: dd:hh:mm:ss.fffffff format, Actual: 0:00:00:04.4786555"
    }
  ]
}
```

Related: [Fixing TimeSpan formatting to match API expectations](https://github.com/Microsoft/ApplicationInsights-Ruby/pull/19)

**Comment by SergeyKanzhelev** [comment]
@OsvaldoRosado you've been fixing issues around this formatting recently. Would you please update the schema for the endpoint and regenerate it here.

**Comment by OsvaldoRosado** [comment]
Schema should be up-to-date for the endpoint (and the invalid error message has been fixed awaiting deployment). I'll send a PR to regenerate the copy here

**Comment by OsvaldoRosado** [comment]
Fixed with 804a33e
